storage/test/test_optimistic_kv_store.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from plenum.common.util import randomString
+from storage.optimistic_kv_store import OptimisticKVStore
+
+
+@pytest.fixture()
+def optimistic_store(parametrised_storage) -> OptimisticKVStore:
+    store = OptimisticKVStore(parametrised_storage)
+    return store
+
+
+def gen_data(num):
+    return {randomString(32).encode(): randomString(100).encode()
+            for _ in range(num)}
+
+
+def test_set_get_reject_keys_basic(optimistic_store):
```

Comment:
Can we split this test into multiple test cases? It looks too long and complicated.
We should have individual test for each API method in OptimisticKVStore, plus some integration test cases (multiple operations).

Reply:
I can add some other basic tests if you need, but these tests are building a scenario, they add some keys and check for before and after commit, and then with same keys again